src/sandstorm/supervisor.capnp

```diff
@@ -274,15 +256,28 @@ struct ApiTokenOwner {
 
       introducerIdentity @9 :Text;
       # The identity ID through which a user's powerbox action caused the grain to receive this
-      # token. This is the identity against which the `requiredPermissions` parameter
-      # to `restore()` will be checked. This field is only intended to be filled in by the
-      # front-end during a powerbox request; a regular `save()` call produces a capability that
-      # has no "introducer".
+      # token. The token will become invalid if this identity loses access to the grain. This field
```

Comment:
> Is there a good reason not to declare `grain.introducerIdentity` entirely obsolete and ignore it?

It would be nice if we could continue to support existing grain-owned tokens. Most of these will have an introducer identity and therefore should be auto-revoked if that identity loses access to the grain. To accomplish this, we could either migrate these tokens so that the data in `introducerIdentity` field moves into the `requirements` field, or we could follow my strategy here, which is to continue honoring the `grain.introducerIdentity` field if it is present.